packages/storage/src/__tests__/queryExtensions.test.ts

```diff
@@ -20,10 +20,11 @@ vi.mock('node:fs/promises', () => ({
       return 'MOCKED_FILE_CONTENT'
     }),
     copyFile: vi.fn(),
+    mkdir: vi.fn(),
```

Comment:
@dthyresson This was all that was needed 🙂

Reply:
ah. thanks. tobbAI > Cursor AI :)